arrow-libs/optics/arrow-optics-ksp-plugin/src/main/kotlin/arrow/optics/plugin/internals/domain.kt

```diff
@@ -4,11 +4,10 @@ import com.google.devtools.ksp.symbol.KSClassDeclaration
 import com.google.devtools.ksp.symbol.KSName
 import java.util.Locale
 
-data class ADT(val pckg: KSName, val el: KSClassDeclaration, val targets: List<Target>) {
-  val sourceClassName = el.qualifiedName?.asString() ?: el.simpleName
-  val sourceName = el.simpleName.asString().replaceFirstChar { it.lowercase(Locale.getDefault()) }
-  val simpleName = el.simpleName.asString()
-  val packageName = pckg.asString()
```

Comment:
Can we keep these names the same? This is our own domain.
This introduces a lot of unnecessary changes in the PR.

Reply:
Sure